src/contexts/TodosProvider.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useMemo, useReducer } from 'react';
+import { Todo } from '../types/Todo';
+import { TodoAction } from '../types/TodoAction';
+import { FilterOptions } from '../types/FilterOptions';
+import { useLocalStorage } from '../hooks/useLocalStorage';
+
+type Props = {
```

Comment:
```suggestion
interface Props  {
```
use interface prefer